src/components/bubble-divider/bubble-divider.tsx

```diff
@@ -21,36 +24,46 @@ export const BubbleDivider = ({
 
   return (
     <div className={`c-bubble-divider-container ${className}`}>
-      {isOpen && getChildren()}
-      <div className={`c-bubble-divider ${variant}`}>
+      <AnimatePresence>{isOpen && getChildren()}</AnimatePresence>
+      <div className={`c-bubble-divider ${variantColor}`}>
         <hr />
-        <Button className="c-bubble-button" onClick={handleBubbleClick}>
+        <Button className={`c-bubble-button ${variantType}`} onClick={handleBubbleClick}>
           {label}
-          {isDropDown() && getArrow()}
+          {variantType === 'drop-down' && getArrow()}
         </Button>
+        <hr />
```

Comment:
Any reason we opted for 2 `hr`s instead of the previous strategy of `position: absolute`?

Reply:
I was having trouble with conflicting z-indexes, so this all lies flat